### ISSUE TITLE
There is no easy way to see the versions of nodes that validators are running

### ISSUE THREAD

**Issue by bowenwang1996** [near/near-explorer#644]
## Story

After the recent change, there is no longer an easy way to see the versions of the validator nodes. This could be annoying when there is a protocol upgrade in progress and people want to see how many nodes have upgraded and which nodes have not. While telemetry is purely optional, every validator at this point opts in and also expects the validator page to display information about versions.

**Comment by frol** [comment]
@bowenwang1996 Do you mean it takes more time due to a need to open the details or you missed the details block in the first place?

![image](https://user-images.githubusercontent.com/304265/120007568-6d125780-bfb0-11eb-80ab-6bb3a8d607a9.png)

I feel like we abuse the page for no good reason. If we focus on the problem, we may arrive to a different solution.

The problem: we need to know how the protocol version upgrade is progressing.

Using telemetry and agent version is an indirect measure to estimate protocol versions. If there is no other way to push the information about the protocol version but through telemetry, then we should at least push the protocol version as a separate field to telemetry and expose it as a metric like "50% of validator seats are ready to migrate to protocol version X"

What do you think?

**Comment by bowenwang1996** [comment]
> Using telemetry and agent version is an indirect measure to estimate protocol versions. If there is no other way to push the information about the protocol version but through telemetry, then we should at least push the protocol version as a separate field to telemetry and expose it as a metric like "50% of validator seats are ready to migrate to protocol version X"
> 
> What do you think?

Yeah I think that is a good idea!

**Comment by frol** [comment]
Closing in favor of #444. This is still blocked on nearcore side https://github.com/near/nearcore/issues/4355
